client/src/pages/PostForm/AskForm/AskForm.component.tsx

```diff
@@ -181,7 +181,6 @@ const AskForm = ({
                 rules={{ validate: areTagsValid }}
                 render={({ field }) => (
                   <Select
-                    inputRef={field.ref}
```

Comment:
?

Reply:
React-select does not want us to access its internals anymore:
`NOTE: Accessing any of the internals of the Select component using refs is not part of our public API. The internals of the Select component can change at any time (including in minor and patch releases). The only methods on the Select component that are part of the public API are the focus() and blur() methods.`
https://react-select.com/upgrade

Reply:
The inputRef allows `react-hook-form` to focus the error input. But this is not being used as we have a separate `alert` component when there are any errors in the `select` component.
https://react-hook-form.com/api/usecontroller/controller